deployment.yaml

```diff
@@ -71,9 +74,16 @@ spec:
             readOnly: true
           - name: cloudsql
             mountPath: /cloudsql
+      - name: keycloak-metrics
+        image: akvo/prometheus-filesystem-exporter:0.3.1
```

Comment:
Where is this container created and published?

Reply:
public Docker registry